tests/test_models.py

```
@@ -111,3 +115,372 @@ def test_get_content(self, app):
+            db.session.commit()
+
+            # Verify cascade deletion of variants
+            assert SkillVariant.query.filter_by(skill_id=skill_id).count() == 0
+
+            # Verify category still exists
+            assert SkillCategory.query.get(category_id) is not None
+
+    def test_user_with_multiple_skills(self, app, test_data_builder):
+        """Test user with multiple skills"""
+        with app.app_context():
+            user_data = test_data_builder.create_user_with_skills(skill_count=5)
```

Ruff (0.13.1)

Local variable `user_data` is assigned to but never used

Remove assignment to unused variable `user_data`

(F841)